src/format/parser.d

```diff
@@ -2503,6 +2511,18 @@ private:
 		assert(0);
 	}
 
+	void parseShortenedFunctionBody() in(match(TokenType.FatArrow)) {
+		auto spanGuard = span!Span();
+		split();
+		nextToken();
+		split(true);
+		space();
+		parseExpression();
+		split();
+		nextToken();
```

Comment:
You might want to check it is the token you expect.

Reply:
What should it do if it's not? Nothing, then return?

Reply:
Yes, pretty much.